src/routes/+page.svelte.md

```diff
@@ -85,6 +85,32 @@ In their attempt to get Asahi Linux out (and support Apple Silicon) as soon as p
 
 Still though, that's besides the main problem. It was [only about 3 days ago this video came out](https://www.youtube.com/watch?v=k0cnMUroMlQ), in which a DE is working for the first time. Not only is it still in a state far from prime-time, but compounded with the tweet above they didn't even bother trying to speak with the devs of the project about its current state. 
 
+### Shipping third-party Firefox theme by default on Manjaro GNOME
```

Comment:
this could be broken out into its own heading of "ignoring upstream" alongside asahi? i'm not sure though